### DISCUSSION TITLE
Is there a way customize content of Confirm email, Reset Password emails?

### DISCUSSION THREAD

**Discussion by jaypeemayo** [OrchardCMS/OrchardCore#10790] [Q&A] [answered]
I want to add an additional message to the email sent to the user because at the moment the email is so simple
![image](https://user-images.githubusercontent.com/21273774/144366824-87f5a0fa-610c-49a8-bc0a-43bdae411dcf.png)

Is there a way to add additional info to the email body? please advice.

**Comment by ns8482e** [comment, marked as answer]
- For Confirm email - override template`TemplateUserConfirmEmail`
- For Forgot password email - override template `TemplateUserLostPassword`

**Reply by jaypeemayo** [reply]
Hi , would you mind providing me an example on how to override? @ns8482e

**Reply by ns8482e** [reply]
Create `Views\TemplateUserConfirmEmail.cshtml` in your Web project

**Reply by jaypeemayo** [reply]
nice this works
